torch_geometric/nn/models/neural_fingerprint.py

```diff
@@ -0,0 +1,73 @@
+import torch
+import torch.nn as nn
+
+from torch_geometric.nn import MessagePassing
+from torch_geometric.utils import add_self_loops
+
+
+class NeuralConv(MessagePassing):
+    def __init__(self, num_features, fingerprint_length):
```

Comment:
add parameter max_degree

Reply:
+1. Add parameter `max_degree` and set default value to 5.